### DISCUSSION TITLE
Stats server error in docker logs

### DISCUSSION THREAD

**Discussion by Neoblock8** [blockscout/blockscout#10991] [Q&A]
Hi,
 While running blockscout in docker-compose, always stats server is restarting and getting this error.
 
![image](https://github.com/user-attachments/assets/a9ee9cd2-ec8c-45b6-817f-ea4477164405)

Where can i add this variable

**Comment by GimluCom** [comment]
You may add the ENV to the [`envs/common-stats.env`](https://github.com/blockscout/blockscout/blob/master/docker-compose/envs/common-stats.env) file.

**Reply by Neoblock8** [reply]
> You may add the ENV to the [`envs/common-stats.env`](https://github.com/blockscout/blockscout/blob/master/docker-compose/envs/common-stats.env) file.

Building a new docker image for frontend, always container will get restarts. Checked frontend logs, but it doesnt return anything
